### PR TITLE
Small changes for qrcode generation and link

### PR DESCRIPTION
**Pull request by dalgaaf**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
       <div class="slides">
         <section id="qrcode" data-menu-title="QR code">
           <div class="qrcode" id="qrcode-talk"/>
-          <p><a href="https://example.com/link/to/this/talk" target="_blank"
+          <p><a style="white-space: nowrap" href="https://example.com/link/to/this/talk" target="_blank"
```
Comment (fghaas) [inline index.html]: Could you please put that into the SCSS file, rather than using an inline `style` attribute?

File: index.html [hunk 2]
```diff
@@ -68,6 +68,8 @@
          * it */
         if (target) {
             var qr = new QRCode(target, {
+                width : 500,
```
Comment (fghaas) [inline index.html:71]: Is there a particular reason to hardcode this in pixels? Does qrcode.js not support `vh` and `vw` units?

File: index.html (hunk 1 above)
Comment (dalgaaf) [reply index.html]: moved it to scss file and rebased.

File: index.html (hunk 2 above)
Comment (dalgaaf) [reply index.html:71]: The problem is here: without hardcoded pixels the qrcode looks really blurry on my presentations. I tried to make use of useSVG instead, but SVG is buggy in qrcode (needs a fix in the project) and I didn't manage to get the link on the page with inline svg for some reasons. I could try again later to investigate further.

Comment (fghaas) [reply index.html:71]: That's odd; I've never had this issue. Which browser are you using?

File: index.html (hunk 1 above)
Comment (fghaas) [reply index.html]: Cool, thanks!

File: index.html (hunk 2 above)
Comment (dalgaaf) [reply index.html:71]: chromium or opera for example.

Check these screenshots from the same resolution: https://drive.google.com/file/d/0BxjHfqaf64A7S2RRRWNlUko3QTQ/view?usp=sharing
https://drive.google.com/file/d/0BxjHfqaf64A7UHQxYVhKcFVLMnc/view?usp=sharing

Comment (fghaas) [reply index.html:71]: Oh, I see now. Sure, makes sense.

**Pull request merged**